.github/workflows/e2e-check-safe-apps.yml

```diff
@@ -0,0 +1,26 @@
+name: Run e2e test for check safe apps liveness
+on:
+  pull_request:
```

Comment:
We should remember to remove this when it's ready to avoid running on each commit of PR :wink:

Reply:
yep, that's for testing purposes while developing

Reply:
btw i'm not able to make the schedule work by now 😩 and i think is configured properly

Reply:
Don't worry. Events different from `pull_request` need the action to be already on the main branch of the repo. That way you can avoid people adding PRs with cron actions overflowing your capacity.

Once this action reaches the main branch it will work for sure

Reply:
Ahh awesome. I was guessing that but unsure 😆